src/importexport/guitarpro/internal/gtp/gpconverter.h

```diff
@@ -251,6 +251,40 @@ class GPConverter
     std::vector<SLine*> m_trillElements;
 
     std::map<uint16_t, uint16_t > _drumExtension;
+    std::unordered_map<int, int> keyToSharpsConverter{
```

Comment:
can be static in cpp
plus, should be m_ keyToSharpsConverter
feel free to change the neighbours' names if they don't match the style =)